src/dynamo/util.ts

```diff
@@ -106,41 +120,38 @@ const createClient = (config: ClientConfig): ClientInterface => {
         });
       }
 
-      return client.createTable(params).promise();
+      return client.createTable(params);
     },
-    get: async (id: string): Promise<DocumentClient.GetItemOutput> => {
+    get: async (id: string): Promise<GetCommandOutput> => {
       if (!id) {
         throw new Error(`Error getting item from table ${config.name} with, id is null`);
       }
 
       try {
-        const result = await documentClient
-          .get({
-            TableName: tableName(config.name),
-            Key: {
-              [config.partitionKey]: id,
-            },
-          })
-          .promise();
+        const result = await documentClient.get({
+          TableName: tableName(config.name),
+          Key: {
+            [config.partitionKey]: id,
+          },
+        });
 
         return result;
       } catch (error: any) {
         throw new Error(`Error getting item from table ${config.name} with id ${id}: ${error.message}`);
       }
     },
-    scan: async (params: Omit<DocumentClient.ScanInput, 'TableName'>): Promise<DocumentClient.ScanOutput> => {
+    scan: async (params: Omit<ScanCommandInput, 'TableName'>): Promise<ScanCommandOutput> => {
       try {
-        return await documentClient
-          .scan({
-            TableName: tableName(config.name),
-            ...params,
-          })
-          .promise();
+        return await documentClient.scan({
+          TableName: tableName(config.name),
+          ...params,
+          ExclusiveStartKey: params.ExclusiveStartKey || undefined,
```

Comment:
It did not like when ExclusiveStartKey is null, such as when on the first page of a list